.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name: Main build
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+
+jobs:
+    build-gradle:
+        name: build and Test Gradle 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'temurin'

```

Comment:
Hi apologies for not noticing this earlier but can you please change this to 'semeru' as thats the distribution we use

Reply:
fixed